app/controllers/sidebar_sections_controller.rb

```diff
@@ -18,25 +20,32 @@ def create
       SidebarSection.create!(section_params.merge(sidebar_urls_attributes: links_params))
 
     if sidebar_section.public?
-      StaffActionLogger.new(current_user).log_create_public_sidebar_section(sidebar_section)
+      StaffActionLogger.new(current_user).log_create_public_sidebar_section(
+        sidebar_section.preload(:sidebar_urls),
+      )
+
       MessageBus.publish("/refresh-sidebar-sections", nil)
       Site.clear_anon_cache!
     end
 
-    render json: SidebarSectionSerializer.new(sidebar_section)
+    render_serialized_sidebar_section(sidebar_section)
   rescue ActiveRecord::RecordInvalid => e
     render_json_error(e.record.errors.full_messages.first)
   end
 
   def update
     sidebar_section = SidebarSection.find_by(id: section_params["id"])
+
     @guardian.ensure_can_edit!(sidebar_section)
 
     ActiveRecord::Base.transaction do
       sidebar_section.update!(section_params.merge(sidebar_urls_attributes: links_params))
       sidebar_section.sidebar_section_links.update_all(user_id: sidebar_section.user_id)
+      sidebar_section.preload([sidebar_section_links: :linkable])
```

Comment:
Yeah really seems like we could just use `sidebar_section.includes(sidebar_section_links: :linkable)` here and other places we used `preload`, no?